NitroxServer-Subnautica/GameLogic/Entities/Spawning/ReefbackBootstrapper.cs

```diff
@@ -23,11 +28,10 @@ public void Prepare(Entity parentEntity, DeterministicBatchGenerator determinist
         {
             for(int spawnPointCounter = 0; spawnPointCounter < LocalCreatureSpawnPoints.Count; spawnPointCounter++)
             {
-                Vector3 localSpawnPosition = LocalCreatureSpawnPoints[spawnPointCounter];
                 float targetProbabilitySum = (float)deterministicBatchGenerator.NextDouble() * creatureProbabiltySum;
                 float probabilitySum = 0;
 
-                foreach (ReefbackEntity creature in SpawnableCreatures)
+                foreach (ReefbackSpawnData.ReefbackCreature creature in SpawnableCreatures)
```

Comment:
Let's refactor a bit to get two methods: `SpawnReefbackPlants` and `SpawnReefbackAnimals`